jobs/run_travissync.groovy

```diff
@@ -0,0 +1,33 @@
+import util.Common
+Common.makeFolders(this)
+
+pipelineJob('travissync/travissync') {
```

Comment:
Do we need a dedicated top level folder for this job?  People are complaining about too much clutter at the top level so I'd prefer to at least us a generic name.